ttnn/cpp/ttnn/operations/loss/loss_pybind.hpp

```diff
@@ -0,0 +1,131 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <pybind11/operators.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "loss.hpp"
+#include "loss_types.hpp"
+#include "ttnn/cpp/pybind11/export_enum.hpp"
+
+namespace py = pybind11;
+
+namespace ttnn::operations::loss {
+
+namespace detail {
+
+void bind_loss_type(py::module& m) {
+
+    export_enum<LossReductionMode>(m, "LossReductionMode");
+}
+
+void bind_mse_loss_function(py::module& module) {
```

Comment:
These functions don't even need to be in the .hpp because nobody will call them externally. 
So they can be defined in cpp

- bind_loss_type
- bind_mse_loss_function
- bind_mae_loss_function

Reply:
yes. moved to cpp